.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -259,7 +268,36 @@ main() {
     exit 1
   fi
 
-  run_tests
+  run_tests "${NAME_SPACE}"
+}
+
+main_rbac() {
+  configure_namespace ${NAME_SPACE_RBAC}
+  installPipelinesOperator "${DIR}"
+  install_helm
+  uninstall_helmchart
+
+  cd "${DIR}"
+  apply_yaml_files "${DIR}" ${NAME_SPACE_RBAC}
+  add_helm_repos
+
+  echo "Tag name : ${TAG_NAME}"
+
+  helm upgrade -i ${NAME_SPACE_RBAC} -n ${NAME_SPACE_RBAC} rhdh-chart/backstage --version ${CHART_VERSION} -f $DIR/value_files/${HELM_CHART_VALUE_FILE_NAME} --set global.clusterRouterBase=${K8S_CLUSTER_ROUTER_BASE} --set upstream.backstage.image.tag=${TAG_NAME}
+
+  check_backstage_running ${NAME_SPACE_RBAC}
+  backstage_status=$?
+
+  echo "Display pods for verification..."
+  oc get pods -n ${NAME_SPACE_RBAC}
+
+  if [ $backstage_status -ne 0 ]; then
+    echo "Backstage is not running. Exiting..."
+    exit 1
+  fi
+
+  run_tests ${NAME_SPACE_RBAC}
 }
 
 main
+main_rbac
```

Comment:
Deploying RHDH first and waiting for its running status takes time. Since this approach requires another deployment for RBAC, it doubles the waiting time.

A suggested improvement is to create a separate Helm deployment function for regular tests, then immediately initiate a deployment for RBAC without waiting/checking for the previous deployments running state. we can check the running status in `main` and `main_rbac`, as they do now.